packages/vitest/src/node/core.ts

```diff
@@ -122,8 +122,8 @@ export class Vitest {
         this.console.error(c.dim('filter:  ') + c.yellow(filters.join(comma)))
       if (this.config.include)
         this.console.error(c.dim('include: ') + c.yellow(this.config.include.join(comma)))
-      if (this.config.watchIgnore)
-        this.console.error(c.dim('ignore:  ') + c.yellow(this.config.watchIgnore.join(comma)))
+      if (this.config.watchExclude)
+        this.console.error(c.dim('ignore:  ') + c.yellow(this.config.watchExclude.join(comma)))
```

Comment:
```suggestion
        this.console.error(c.dim('exclude:  ') + c.yellow(this.config.watchExclude.join(comma)))
```

Reply:
I think `exclude` is a bit misleading. Renamed to `watch exclude`